debian/control

```diff
@@ -1,23 +1,24 @@
 Source: noderedrevpinodes-server
 Section: misc
 Priority: optional
-Maintainer: Philipp Rosenberger <p.rosenberger@kunbus.com>
-Build-Depends: debhelper (>= 12)
-Standards-Version: 4.6.0
+Maintainer: KUNBUS GmbH <support@kunbus.com>
+Rules-Requires-Root: no
 Homepage: https://github.com/erminas/noderedrevpinodes-server
 Vcs-Browser: https://github.com/RevolutionPi/noderedrevpinodes-server
 Vcs-Git: https://github.com/RevolutionPi/noderedrevpinodes-server.git
+Build-Depends: debhelper-compat (= 12), dh-python, python3
+Standards-Version: 4.3.0
 
 Package: noderedrevpinodes-server
 Architecture: all
 Depends:
  ${misc:Depends},
- python3 (>= 3.5),
+ ${python3:Depends},
  python3-revpimodio2 (>= 2.5.1),
  python3-bcrypt (>= 3.1.2),
  python3-cryptography (>= 1.7.1),
  python3-websockets(>= 3.2-2),
- python3-distro(>=1.0.0)
+ python3-distro(>= 1.0.0)
```

Comment:
Shouldn't we add a reformat commit, because we use at least two spaces as indention in our other packages?

Reply:
I had to laugh... well played